### ISSUE TITLE
Long wait time after clicking `power off` in `gdm`

### ISSUE THREAD

**Issue by lebensterben** [clearlinux/distribution#1940]
After upgrading to `GDM 3.34`, I observe a long wait time (> 10 seconds) after I click the power off button.

This system is still responsive and everything is working well, but IMO, the expected behaviour should be a pop-up window instantly shows up to ask you whether to power off, to reboot, or to cancel.

I'm not sure whether this is an upstream bug, or a bug caused by recent deprecation of certain packages.

**Comment by heterotic** [comment]
There is some dependency on starting PackageKit now

https://github.com/GNOME/gnome-shell/commit/83c6b2ab48f9a2321e4371d276958d0643e517e4

which means waiting for the D-Bus service to time out. Without implementing support for swupd it's somewhat useless. Adding `--backend dummy` to or even just masking the corresponding systemd unit speeds it up and doesn't appear to break anything.

**Comment by mrkz** [comment]
@lebensterben Thanks for your report.

As @heterotic mentions, this is due to PackageKit getting in the middle, I noted this when I started the GNOME update and as we don't use PackageKit whatsoever, there were [changes](https://github.com/clearlinux/clr-bundles/commit/3ad5b6e81d62ad130731d9983ff739fa42059438) in the desktop bundle to drop it from it and got successful results in my mix outputs, however, there were some packages that still either `Requires` or `BuildRequires` it.

I pushed new build without those requirements, so we should see that behavior going away (apparently for release **33010**).

Let me know if this is still appearing after such release.
